lib/events/dynamoevents/dynamoevents_test.go

```diff
@@ -75,7 +75,7 @@ func (s *DynamoeventsSuite) SetUpSuite(c *check.C) {
 	s.log = log
 	s.EventsSuite.Log = log
 	s.EventsSuite.Clock = fakeClock
-	s.EventsSuite.QueryDelay = time.Second
+	s.EventsSuite.QueryDelay = time.Second * 30
```

Comment:
30s sleep time in tests is quite big value. Can we consider switching to backoff call ?

Reply:
Pushed a fix that backs off on the test causing problems and reduces the query delay to 5s.